classes/npm.bbclass

```diff
@@ -239,7 +239,8 @@ do_install() {
 
 do_prepare_build_append() {
     INSTALL_FLAGS="--offline --only=production --no-package-lock --verbose \
-                   --arch=${NPM_ARCH} --target_arch=${NPM_ARCH}"
+                   --arch=${NPM_ARCH} --target_arch=${NPM_ARCH} \
+                   --no-audit"
```

Comment:
Not against it, but can you quantify the gain, roughly?

Reply:
On a slow FS I hat up to 25% time reduction on NPM rebuilds.